doc/nrf/releases/release-notes-latest.rst

```diff
@@ -93,4 +93,11 @@ The following list summarizes the most important changes inherited from upstream
 
 The following list contains |NCS| specific additions:
 
-*
+* The Bluetooth controllers default uses split architecture
+* NCS Partition Manager (PM) for flash partitioning
+* NCS network socket and address extensions:
+  AF_LTE, NPROTO_AT, NPROTO_PDN, NPROTO_DFU, SOCK_MGMT, SO_RCVTIMEO, SO_BINDTODEVICE, SOL_PDN, SOL_DFU, SO_PDN_CONTEXT_ID, SO_PDN_STATE, SOL_DFU, SO_DFU_ERROR, TLS_SESSION_CACHE,
+  SO_SNDTIMEO, MSG_TRUNC
+  SO_SILENCE_ALL, SO_IP_ECHO_REPLY, SO_IPV6_ECHO_REPLY
```

Comment:
```suggestion
* Added the following network socket and address extensions to the :ref:`zephyr:bsd_sockets_interface` interface to support the functionality provided by the :ref:`nrfxlib:bsdlib`:
  AF_LTE, NPROTO_AT, NPROTO_PDN, NPROTO_DFU, SOCK_MGMT, SO_RCVTIMEO, SO_BINDTODEVICE, SOL_PDN, SOL_DFU, SO_PDN_CONTEXT_ID, SO_PDN_STATE, SOL_DFU, SO_DFU_ERROR, TLS_SESSION_CACHE, SO_SNDTIMEO, MSG_TRUNC, SO_SILENCE_ALL, SO_IP_ECHO_REPLY, SO_IPV6_ECHO_REPLY
```
Is this what they are needed for?

Reply:
Those are the exact extensions added.
Listing them here makes it much easier to search for them, if they are not already know to the reader.
(some are nordic specific)

Reply:
>  Listing them here makes it much easier to search for them, if they are not already know to the reader.

But how can we avoid this going stale?